### PR TITLE
feat: install debugger

### PR DESCRIPTION
**Pull request by andyMrtnzP**

### CODE REVIEW

**Comment by andyMrtnzP** [comment]
To be merged after https://github.com/browserless/debugger/pull/26 is in the npm registry

File: package.json [hunk 1]
```diff
@@ -62,7 +64,8 @@
     "puppeteer-extra": "^3.3.6",
     "puppeteer-extra-plugin-stealth": "^2.11.2",
     "queue": "^7.0.0",
-    "systeminformation": "^5.22.7"
+    "systeminformation": "^5.22.7",
```
Comment (joelgriffith) [inline package.json:67]: These should probably go into `optionalDependencies` as they're build related, correct?

Comment (andyMrtnzP) [reply package.json:67]: That lib is used for getting the machine stats 🤔: https://github.com/browserless/browserless/blob/feat/debugger/src/monitoring.ts#L7-L23

**Review by joelgriffith — CHANGES_REQUESTED** [review]
One final thought above!

**Review by joelgriffith — APPROVED** [review]

**Pull request merged**